assets/css/screenplay-container.scss

```diff
@@ -23,7 +22,7 @@
       background-color: $cool-gray-15;
       padding: 24px 20px;
       border-radius: 16px 0px 0px 0px;
-      flex-grow: 1;
+      height: 100%;
```

Comment:
To be more explicit I think this could be `flex-basis: 100%`. (`flex-basis` defaults to `auto` which means to use any defined width/height as the basis, so it's one less layer of indirection)

Reply:
So I was able to get this to work, but it requires other styling changes in other files (specifically needs changes to `.pending-screens-page`). I think I'll leave it alone for now unless you prefer making those changes.

Reply:
Interesting. Well, if this works, I don't see anything wrong with it.